cnn_framework/test_data/ensemble_predictor/predictor_input.py

```diff
@@ -0,0 +1,11 @@
+predictor_model(embedding_size=512, depth=3, scale_output=0.05,
```

Comment:
The formatting in this file is weird. Can you fix it?

Reply:
Done